Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/macros.py

```diff
@@ -10,6 +10,76 @@
 # QUA macros #
 ##############
 
+# Macro for performing a single shot active reset.
+def active_reset_single(threshold=None, Ig=None):
```

Comment:
Maybe just `active_reset`, without the single?

Reply:
Actually, since the `active_reset_until_success `function does the same thing with max_tries=1, I am thinking about removing it completely, what do you think?

Reply:
See the message I just sent you over Slack :)

But in any case, I would keep it. we can just have the `active_reset` function just call the `active_reset_until_sucess` with max_tries=1. But it is more user-friendly to have both.

Reply:
Alternatively, we can just have a single function called `active_reset` and that it'll have an optional max_tries parameter (default would be 1)

Reply:
Yes that is what I meant, just one active reset which is `active_reset_until_success(max_tries=1)`
I will wrap it up as discussed on Slack